.gitattributes

```
@@ -0,0 +1,10 @@
+/docs/* export-ignore
+/.github export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php-cs-fixer.php export-ignore
+/phpstan.neon export-ignore
+/phpunit.xml.dist export-ignore
+/run-tests.sh export-ignore
+/tests export-ignore
+/wait-for-it.sh export-ignore
```

These files don't exist.

```suggestion
/tests export-ignore
```